### DISCUSSION TITLE
Annotations to Draw on the Remote Screen Share

### DISCUSSION THREAD

**Discussion by satishbabariya** [aryan02420/DrawOnScreen#4] [Q&A]
@aryan02420 Can you share any ideas of implementation regarding annotation screen of remote user while user shared his screen over webrtc, 
ref: [Use annotation while sharing your screen in Microsoft Teams](https://support.microsoft.com/en-au/office/use-annotation-while-sharing-your-screen-in-microsoft-teams-876ba527-7112-437e-b410-5aec7363c473 )

**Comment by aryan02420** [comment]
@satishbabariya 
Certainly! This was initially a part of my plans. However, at the moment, I mainly use it as a simple annotation tool for screenshots. I do intend to explore options for sharing and drawing over remote screens.

You might also want to consider looking into alternative tools that support screen sharing. I have used [Deskreen](https://github.com/pavlobu/deskreen), but I don't think it currently supports annotation.
